source/manage-apps/secure.txt

```diff
@@ -236,18 +236,22 @@ CSV
 
 .. 
    Note to docs writer: the hosted csv and json files are managed in the
-   docs-realm-sdks repo: https://github.com/10gen/docs-realm-sdks
+   docs-realm-sdks repo: https://github.com/10gen/docs-realm-sdks   
 
-.. note::
+Device Sync
+^^^^^^^^^^^
 
-   The above IP list only applies to outgoing requests from {+service-short+} functions,
-   including triggers and HTTPS endpoints that make outgoing requests.
+The above IP list only applies to outgoing requests from {+service-short+} functions,
+including triggers and HTTPS endpoints that make outgoing requests.
 
-   For requests that originate from the Sync server, we recommend allowlisting
-   the entire subnet of the app's Deployment Region and cloud provider. You can
-   find the Deployment Region in the {+ui+} under :guilabel:`App Settings >
-   General > Deployment Region`.
-   
+For requests that originate from the Sync server, we recommend allowlisting
+the entire subnet of the app's Deployment Region and cloud provider. You can
```

Comment:
which app? the app services app or the customer's app?

Reply:
It's the App Services App - the next line talks about how to find the Deployment Region in the App Services settings. But, to your point, I've just pushed an update to capitalize this.